docs/install/operator/configuring-serving-cr.md

```diff
@@ -0,0 +1,393 @@
+  knative-ingress-gateway:
+    selector:
+      custom: ingressgateway
+  config:
+    istio:
+      gateway.knative-serving.knative-ingress-gateway: "custom-ingressgateway.istio-system.svc.cluster.local"
+```
+
+The key in `spec.config.istio` is in the format of `gateway.{{gateway_namespace}}.{{gateway_name}}`.
+
+## Configuration of cluster local gateway:
+
+Update `spec.cluster-local-gateway` to select the labels of the new cluster-local ingress gateway:
+
+### Default local gateway name:
+
+Go through the guide [here](https://knative.dev/development/install/installing-istio/#updating-your-install-to-use-cluster-local-gateway) to use local cluster gateway,
+if you use the default gateway called `cluster-local-gateway`.
+
+### Non-default local gateway name:
+
+If you create custom local gateway with a name other than `cluster-local-gateway`, update `config.istio` and the
+`cluster-local-gateway` selector:
+
+This example shows a service and deployment `custom-local-gateway` in the namespace `istio-system`, with the
+label `custom: custom-local-gw`:
+
+```
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  cluster-local-gateway:
+    selector:
+      custom: custom-local-gateway
+  config:
+    istio:
+      local-gateway.knative-serving.cluster-local-gateway: "custom-local-gateway.istio-system.svc.cluster.local"
+```
+
+## High availability:
+
+By default, Knative Serving runs a single instance of each controller. The `spec.high-availability` field allows you to configure the number of replicas for the following master-elected controllers: `controller`, `autoscaler-hpa`, `networking-istio`. This field also configures the `HorizontalPodAutoscaler` resources for the data plane (`activator`):
+
+The following configuration specifies a replica count of 3 for the controllers and a minimum of 3 activators (which may scale higher if needed):
+
+```
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  high-availability:
+    replicas: 3
+```
+
+## Managing resources for containers
+
+The operator custom resource allows you allows you to configure resource requests and limits for the Knative system containers.
+Requests and limits may be configured for the following containers: `activator`, `autoscaler`, `controller`, `webhook`, `autoscaler-hpa`,
+`networking-istio` and `queue-proxy`.
+
+To override resource settings for a specific container, create an entry in the `spec.resources` list with the container name and the [Kubernetes resource settings](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container).
+
+For example, the following KnativeServing resource configures the `activator` to request 0.3 CPU and 100MB of RAM, and sets hard limits of 1 CPU, 250MB RAM, and 4GB of local storage:
+
+\```
```

Comment:
You need to remove the "\" here and on line 359. I can't use suggestion blocks for that, because they use the <code>```</code> shortcode.

Reply:
done